smart-contracts/test/UnlockDiscountToken/governance.js

```diff
@@ -1,166 +1,146 @@
 // tests adapted/imported from https://github.com/OpenZeppelin/openzeppelin-contracts/blob/7e41bf2259950c33e55604015875b7780b6a2e63/test/token/ERC20/extensions/ERC20VotesComp.test.js
-const { network } = require('hardhat')
-const { expect, assert } = require('chai')
+const { ethers } = require('hardhat')
+const { expectRevert, time } = require('@openzeppelin/test-helpers')
+
 const {
-  BN,
+  ADDRESS_ZERO,
   expectEvent,
-  expectRevert,
-  time,
-} = require('@openzeppelin/test-helpers')
-
-const UnlockDiscountTokenV3 = artifacts.require('UnlockDiscountTokenV3.sol')
-const { promisify } = require('util')
-
-const queue = promisify(setImmediate)
-const { ADDRESS_ZERO } = require('../helpers/constants')
-
-async function countPendingTransactions() {
-  return parseInt(
-    await network.provider.send('eth_getBlockTransactionCountByNumber', [
-      'pending',
-    ])
-  )
-}
-
-async function batchInBlock(txs) {
-  try {
-    // disable auto-mining
-    await network.provider.send('evm_setAutomine', [false])
-    // send all transactions
-    const promises = txs.map((fn) => fn())
-    // wait for node to have all pending transactions
-    while (txs.length > (await countPendingTransactions())) {
-      await queue()
-    }
-    // mine one block
-    await network.provider.send('evm_mine')
-    // fetch receipts
-    const receipts = await Promise.all(promises)
-    // Sanity check, all tx should be in the same block
-    const minedBlocks = new Set(
-      receipts.map(({ receipt }) => receipt.blockNumber)
-    )
-    expect(minedBlocks.size).to.equal(1)
+  notExpectEvent,
+  compareBigNumbers,
+  compareBigNumberArrays,
+} = require('../helpers')
 
-    return receipts
-  } finally {
-    // enable auto-mining
-    await network.provider.send('evm_setAutomine', [true])
-  }
```

Comment:
all that logic was intended to test that the gov contract could only add a single endpoint per block. This was adapted from OZ tests and is actually tested already by OZ so we don't need to test it again here as we rely on their work.